proposals/0001-dep-process.md

```diff
@@ -0,0 +1,147 @@
+  likely involve both discussion on the PR comment thread and elsewhere (IRC,
+  etc).
+* Consider drafting or prototyping an implementation to demonstrate your
+  proposal and work out all the details. This step isn't necessary, however: a
+  proposer does not need to be a developer.
+* If consensus seems to have emerged (for or against the proposal), a team
+  member will assign an DEP number, update the status, and merge the PR.
+* Small tweaks (grammar, clarifications) to a merged DEP can take place as
+  regular github PRs; revisiting or significantly revising should take place as
+  a new DEP.
+
+DEPs should have a type ("Standard" or "Process") and a status.
+
+# Reference Documentation
+[reference-documentation]: #reference-documentation
+
+DEPs should have a type:
+
+* **Standard** for technical changes to the protocol, on-disk formats, or
+  public APIs.
+* **Process** for formalizing community processes or other (technical or
+  non-technical) decisions. For example, a security vulnerability reporting
+  policy, a process for handling conflicts of interest, or procedures for
+  mentoring new developers.
+
+The status of an DEP can be:
+
+* **Draft**: writen up, open PR for discussion
+* **Active**: PR accepted; adopted or intended for implementation in mainline
+  libraries and clients as appropriate
+* **Closed**: PR was closed without merging; either consensus was against, a
+  decision was postponed, or the authors withdrew their proposal.
+* **Superseded**: a formerly "active" DEP has been made obsolete by a new
+  active DEP; the new DEP should specify specific old DEPs that it would
+  supersede.
+
+A changelog should be kept in the DEP itself giving the date of any changes of
+status.
+
+A template file is provided, but sections can be added or removed as
+appropriate for a specific DEP.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+There are already multiple sources of technical documentation: the Dat
+[protocol website][proto-website], the Dat [whitepaper][whitepaper], Dat
+website [documentation section][docs], the [discussion repo][discussion-repo]
+issues, and the [datprotocol github group][datproto-group] (containing, eg, the
+`dat.json` repo/spec). Without consensus and consolidation, this would be "yet
+another" place to look.
+
+[proto-website]: https://www.datprotocol.com/
+[whitepaper]: https://github.com/datproject/docs/blob/master/papers/dat-paper.md
+[docs]: https://docs.datproject.org/
+[datproto-group]: https://github.com/datprotocol
+
+# Rationale and alternatives
+[alternatives]: #alternatives
+
+**TODO:**
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+Who are "core developers"? What is the specific decision making process for
+accepting or rejecting a given DEP? Optimistically, it would be clear from
+reading a PR discussion thread whether "consensus" has been reached or not, but
+this might be ambiguous or intimidating to first-time contributors.
```

Comment:
I propose we create a process whereby we establish "repo admins." These admins would be re-evaluated every N months (I propose 6), and being selected as an admin means committing to involvement for the full N months. At the end of the period, new "repo admins" are selected by unanimous consent of the previous period's admins, and that may mean no change in the admins. I would suggest one admin per organization which has an interest in Dat (which at this point would be Code for Science and Society, Blue Link Labs, and the Internet Archive) but there's no reason to follow that exactly.

For decision-making, I would also suggest that any decision requires unanimous consent from the repo admins, with an informal requirement that we factor community feedback in the decision.

Reply:
This is a reasonable proposal. I think I want to research this governance aspect of the prospect a bit more... I will revise the other aspects, and maybe we can merge this as a draft to separate the desire to move ahead from committing to all the governance details?

Reply:
This article had a lot of very specific recommendations/proposals for "building cooperation into the system" that may be worth considering: https://medium.com/@nayafia/the-problem-with-voting-8cff39f771e8